Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:3.6-alpine AS build-image
+
+ENV PYTHONDONTWRITEBYTECODE 1
+
+WORKDIR /create-github-release
+
+COPY . /create-github-release/
+
+RUN apk update
+RUN apk add git libmagic
+RUN pip install --upgrade pip
+RUN pip install pybuilder==0.11.17
+RUN pyb install_dependencies
+RUN pyb
+RUN pyb publish
+
+FROM python:3.6-alpine
+
+ENV PYTHONDONTWRITEBYTECODE 1
+
+WORKDIR /opt/create-github-release
+
+COPY --from=build-image /create-github-release/target/dist/ghrelease-*/dist/ghrelease-*.tar.gz /opt/create-github-release
+
+RUN apk update
```

Comment:
Why this apk update? Could you combine this with line #29? apk add --update

Reply:
Yes I will combine the update/add.  The add is required because the Python local dependency (github3api) needs to be cloned and compiled from source, thus the reason for installing the gcc compiler along with other libraries.  

I don't believe this will be necessary once `github3api` is published to PyPi